redirects/17.0.txt

```diff
@@ -67,9 +67,16 @@ applications/marketing/marketing_automation/getting_started/target_audience.rst
 applications/marketing/marketing_automation/getting_started/testing_running.rst applications/marketing/marketing_automation/testing_running.rst           # getting_started/* --> marketing_automation/*
 applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
 applications/marketing/surveys/time_random.rst applications/marketing/surveys/questions.rst   # time_random --> questions
+<<<<<<< HEAD
```

Comment:
Somehow this ended up in the redirect file, likely left over from resolving a conflict :)
```suggestion
```